lib/fortnox/api/mappers/unit.rb

```diff
@@ -0,0 +1,15 @@
+require "fortnox/api/mappers/base"
+
+module Fortnox
+  module API
+    module Mapper
+      class Unit < Fortnox::API::Mapper::Base
+        KEY_MAP = {}.freeze
+        JSON_ENTITY_WRAPPER = 'Unit'.freeze
+        JSON_COLLECTION_WRAPPER = 'Units'.freeze
+      end
+
+      Registry.register( Unit.canonical_name_sym, Unit )
```

Comment:
Maybe we should move the registry registration to `Fortnox::API::Mapper::Base` to avoid repeating this code in every mapper class?

Reply:
Maybe ... But that makes it more magical :)